.github/CODEOWNERS

```diff
@@ -173,7 +173,7 @@ Tasks/DelayV1/     @rohit-batra  @naga-bandi
 
 Tasks/DeleteFilesV1/   @microsoft/akvelon-build-task-team
 
-Tasks/DeployVisualStudioTestAgentV2/    @nadesu  @ankit-arora12
+Tasks/DeployVisualStudioTestAgentV2/    @nadesu @v-jkarri @ankit-arora12
```

Comment:
DeployVisualStudioTestAgentV2 not own by environment team, you can remove your name

Reply:
Removed my name from DeployVisualStudioTestAgentV2 .